benchmark/metrics.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
```

Comment:
Should these methods be wrapped in a class or object, instead of just in a separate file?

Reply:
Everything has been moved to a module now.